packages/api/test/e2e/api.v36-compat.e2e.ts

```diff
@@ -28,12 +28,8 @@ describe('runtime v36 compatibility', () => {
 
   beforeAll(async () => {
     await cryptoWaitReady();
-    const provider = process.env.TEST_TYPE?.toLowerCase() === 'integration' ? config.wsProvider.integrationV36 : 'ws://localhost:9945';
-    ApiPromise.create({ provider })
-        .then(conn => api = conn)
-        .catch(_ => {
-            console.warn(`>>> ensure a node running v36 runtime is available at ${provider} <<<`)
-        });
+    const provider = process.env.TEST_TYPE?.toLowerCase() === 'integration' ? config.wsProvider.integrationV36 : 'ws://localhost:9944';
+    api = await ApiPromise.create({ provider });
```

Comment:
can you leave an equivalent comment about 'ensure a node running v36 runtime is available at ${provider}' to help debugging local tests

Reply:
Added console.log